lib/rubocop/cop/gusto/ignored_columns_assignment.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Gusto
+      # Enforces proper usage of `ignored_columns` assignment
+      #
+      # This cop ensures that `ignored_columns` is assigned using `+=` with an array
+      # instead of direct assignment, which can cause issues with existing ignored columns.
```

Comment:
How does this sound:
```suggestion
      # instead of direct assignment, which will overwrite the existing  list of ignored columns for the model, or overwrite the list it should inherit in the case of single table inheritance.
```

Reply:
Yes much better thank you